code/datums/uplink/devices and tools.dm

```diff
@@ -89,7 +89,7 @@
 /datum/uplink_item/item/tools/thermal_lens
 	name = "Thermal Imaging Lenses"
 	item_cost = 12
```

Comment:
probably deserves a price decrease since it is now a thermal glasses sidegrade that trades usability for stealth